go.sum

```diff
@@ -521,4 +554,5 @@ gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b h1:h8qDotaEPuJATrMmW04NCwg7v
 gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b/go.mod h1:K4uyk7z7BCEPqu6E+C64Yfv1cQ7kz7rIZviUmN+EgEM=
 honnef.co/go/tools v0.0.0-20190102054323-c2f93a96b099/go.mod h1:rf3lG4BRIbNafJWhAfAdb/ePZxsR/4RtNHQocxwk9r4=
 honnef.co/go/tools v0.0.0-20190523083050-ea95bdfd59fc/go.mod h1:rf3lG4BRIbNafJWhAfAdb/ePZxsR/4RtNHQocxwk9r4=
+rsc.io/pdf v0.1.1/go.mod h1:n8OzWcQ6Sp37PL01nO98y4iUCRdTGarVfzxY20ICaU4=
```

Comment:
Why is this here?

Reply:
```
❯ go mod graph | grep rsc.io/pdf
gonum.org/v1/plot@v0.0.0-20190515093506-e2840ee46a6b rsc.io/pdf@v0.1.1
❯ go mod graph | grep gonum.org/v1/plot@v0.0.0-20190515093506-e2840ee46a6b
gonum.org/v1/gonum@v0.8.2 gonum.org/v1/plot@v0.0.0-20190515093506-e2840ee46a6b
gonum.org/v1/plot@v0.0.0-20190515093506-e2840ee46a6b github.com/ajstarks/svgo@v0.0.0-20180226025133-644b8db467af
gonum.org/v1/plot@v0.0.0-20190515093506-e2840ee46a6b github.com/fogleman/gg@v1.2.1-0.20190220221249-0403632d5b90
gonum.org/v1/plot@v0.0.0-20190515093506-e2840ee46a6b github.com/golang/freetype@v0.0.0-20170609003504-e2365dfdc4a0
gonum.org/v1/plot@v0.0.0-20190515093506-e2840ee46a6b github.com/jung-kurt/gofpdf@v1.0.3-0.20190309125859-24315acbbda5
gonum.org/v1/plot@v0.0.0-20190515093506-e2840ee46a6b golang.org/x/exp@v0.0.0-20180807140117-3d87b88a115f
gonum.org/v1/plot@v0.0.0-20190515093506-e2840ee46a6b golang.org/x/image@v0.0.0-20180708004352-c73c2afc3b81
gonum.org/v1/plot@v0.0.0-20190515093506-e2840ee46a6b gonum.org/v1/gonum@v0.0.0-20180816165407-929014505bf4
gonum.org/v1/plot@v0.0.0-20190515093506-e2840ee46a6b rsc.io/pdf@v0.1.1
❯ go mod graph | grep gonum.org/v1/gonum@v0.8.2
github.com/HdrHistogram/hdrhistogram-go@v1.1.0 gonum.org/v1/gonum@v0.8.2
gonum.org/v1/gonum@v0.8.2 golang.org/x/exp@v0.0.0-20190125153040-c74c464bbbf2
gonum.org/v1/gonum@v0.8.2 golang.org/x/tools@v0.0.0-20190206041539-40960b6deb8e
gonum.org/v1/gonum@v0.8.2 gonum.org/v1/netlib@v0.0.0-20190313105609-8cb42192e0e0
gonum.org/v1/gonum@v0.8.2 gonum.org/v1/plot@v0.0.0-20190515093506-e2840ee46a6b
```

The new histogram library requires it for some reason, but since it's only the mod file I doubt it ever gets used by any code path that Go kit uses.

Reply:
Quick search shows it's only imported in a test: https://github.com/HdrHistogram/hdrhistogram-go/search?q=gonum

God, I hate Go for this.